File: 1020/1020 - Idade em Dias.cs
using System;

class Program
{
    static void Main()
    {
        int age = int.Parse(Console.ReadLine());
        Console.WriteLine($"{age/365} ano(s)");
        age %= 365;
        Console.WriteLine($"{age/30} mes(s)");
        age %= 30;
        Console.WriteLine($"{age} dia(s)");
    }
}

File: 1020/1020 - Idade em Dias.cpp
#include <iostream>
int main() {
    int age;
    std::cin >> age;
    printf("%d ano(s)\n", age/365);
    age %= 365;
    printf("%d mes(es)\n", age/30);
    age %= 30;
    printf("%d dia(s)\n", age);
    return 0;
}
